src/main/java/duke/command/DeleteCommand.java

```diff
@@ -0,0 +1,21 @@
+package duke.command;
+
+import duke.storage.Storage;
+import duke.tasklist.TaskList;
+import duke.ui.Ui;
+
+public class DeleteCommand extends Command {
+
+    private final int taskNo;
```

Comment:
Hey! I think your codes generally look great and I really like how you have followed the coding standard. Maybe just a suggestion for the taskNo, would it be clearer if its taskNum / numTask instead? :))